studio/components/layouts/ProjectLayout/LayoutHeader/FeedbackDropdown/FeedbackWidget.tsx

```
@@ -89,23 +90,27 @@ const FeedbackWidget: FC<Props> = ({
         message: 'Please include a message in your feedback.',
         duration: 4000,
       })
-    } else if (feedback.length > 0) {
+    } else if (feedback.length > 0 && ref !== undefined) {
       setSending(true)
+
       const attachmentUrl = screenshot
         ? await uploadAttachment(ref as string, screenshot)
         : undefined
       const formattedFeedback =
         attachmentUrl !== undefined ? `${feedback}\n\nAttachments:\n${attachmentUrl}` : feedback
-      await post(`${API_URL}/feedback/send`, {
-        message: formattedFeedback,
-        pathname: router.asPath,
-        category: 'Feedback',
-        projectRef: ref,
-        tags: ['dashboard-feedback'],
-      })
-      setSending(false)
-      setFeedback('')
-      ui.setNotification({ category: 'success', message: 'Feedback sent. Thank you!' })
+
+      try {
+        await submitFeedback({
+          projectRef: ref,
+          message: formattedFeedback,
+          pathname: router.asPath,
+        })
+        setFeedback('')
+        ui.setNotification({ category: 'success', message: 'Feedback sent. Thank you!' })
+      } catch (error) {
```

I'm pretty sure you don't need an empty catch block if you've got finally